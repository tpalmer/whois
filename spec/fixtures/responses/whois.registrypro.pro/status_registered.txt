Access to .PRO REGISTRY WHOIS information is provided to assist persons in 
determining the contents of a domain name registration record in the 
.PRO Registry registry database. The data in this record is provided by 
.PRO Registry for informational purposes only, and .PRO Registry does not 
guarantee its accuracy.  This service is intended only for query-based 
access. You agree that you will use this data only for lawful purposes 
and that, under no circumstances will you use this data to: (a) allow, 
enable, or otherwise support the transmission by e-mail, telephone, or 
facsimile of mass unsolicited, commercial advertising or solicitations 
to entities other than the data recipient's own existing customers; or 
(b) enable high volume, automated, electronic processes that send 
queries or data to the systems of Registry Operator, a Registrar, or 
Afilias except as reasonably necessary to register domain names or 
modify existing registrations. All rights reserved. .PRO Registry reserves 
the right to modify these terms at any time. By submitting this query, 
you agree to abide by this policy. 

Domain ID:D6394-PRO
Domain Name:REGISTRYPRO.PRO
Created On:18-Aug-2004 00:00:00 UTC
Last Updated On:14-Jun-2012 21:27:09 UTC
Expiration Date:26-Jan-2017 00:00:00 UTC
Sponsoring Registrar:Corporation Service Company dba CSC Corporate Domains, Inc (R2338-PRO)
Status:CLIENT TRANSFER PROHIBITED
Status:DELETE PROHIBITED
Registrant ID:RSC-1
Registrant Name:Afilias Limited
Registrant Organization:Afilias Limited
Registrant Street1:2 La Touche House
Registrant Street2:IFSC
Registrant Street3:
Registrant City:Dublin
Registrant State/Province:IL
Registrant Postal Code:1
Registrant Country:IE
Registrant Phone:+353.14310511
Registrant Phone Ext.:
Registrant FAX:+353.14310557
Registrant FAX Ext.:
Registrant Email:domainadmin@afilias.info
Admin ID:RSC-1
Admin Name:Afilias Limited
Admin Organization:Afilias Limited
Admin Street1:2 La Touche House
Admin Street2:IFSC
Admin Street3:
Admin City:Dublin
Admin State/Province:IL
Admin Postal Code:1
Admin Country:IE
Admin Phone:+353.14310511
Admin Phone Ext.:
Admin FAX:+353.14310557
Admin FAX Ext.:
Admin Email:domainadmin@afilias.info
Billing ID:RSC-1
Billing Name:Afilias Limited
Billing Organization:Afilias Limited
Billing Street1:2 La Touche House
Billing Street2:IFSC
Billing Street3:
Billing City:Dublin
Billing State/Province:IL
Billing Postal Code:1
Billing Country:IE
Billing Phone:+353.14310511
Billing Phone Ext.:
Billing FAX:+353.14310557
Billing FAX Ext.:
Billing Email:domainadmin@afilias.info
Tech ID:RSC-1
Tech Name:Afilias Limited
Tech Organization:Afilias Limited
Tech Street1:2 La Touche House
Tech Street2:IFSC
Tech Street3:
Tech City:Dublin
Tech State/Province:IL
Tech Postal Code:1
Tech Country:IE
Tech Phone:+353.14310511
Tech Phone Ext.:
Tech FAX:+353.14310557
Tech FAX Ext.:
Tech Email:domainadmin@afilias.info
Right to Use:
Name Server:NS1.AMS1.AFILIAS-NST.INFO
Name Server:NS1.MIA1.AFILIAS-NST.INFO
Name Server:NS1.SEA1.AFILIAS-NST.INFO
Name Server:NS1.YYZ1.AFILIAS-NST.INFO
Name Server:NS1.HKG1.AFILIAS-NST.INFO
Name Server: 
Name Server: 
Name Server: 
Name Server: 
Name Server: 
Name Server: 
Name Server: 
Name Server: 


