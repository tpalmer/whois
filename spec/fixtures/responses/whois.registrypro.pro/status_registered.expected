#disclaimer
  should: %s == "Access to .PRO REGISTRY WHOIS information is provided to assist persons in determining the contents of a domain name registration record in the .PRO Registry registry database. The data in this record is provided by .PRO Registry for informational purposes only, and .PRO Registry does not guarantee its accuracy.  This service is intended only for query-based access. You agree that you will use this data only for lawful purposes and that, under no circumstances will you use this data to: (a) allow, enable, or otherwise support the transmission by e-mail, telephone, or facsimile of mass unsolicited, commercial advertising or solicitations to entities other than the data recipient's own existing customers; or (b) enable high volume, automated, electronic processes that send queries or data to the systems of Registry Operator, a Registrar, or Afilias except as reasonably necessary to register domain names or modify existing registrations. All rights reserved. .PRO Registry reserves the right to modify these terms at any time. By submitting this query, you agree to abide by this policy."


#domain
  should: %s == "registrypro.pro"

#domain_id
  should: %s == "D6394-PRO"


#referral_whois
  should: %s raise_error(Whois::PropertyNotSupported)

#referral_url
  should: %s raise_error(Whois::PropertyNotSupported)


#status
  should: %s == ["CLIENT TRANSFER PROHIBITED", "DELETE PROHIBITED"]

#available?
  should: %s == false

#registered?
  should: %s == true


#created_on
  should: %s CLASS(time)
  should: %s == Time.parse("2004-08-18 00:00:00 UTC")

#updated_on
  should: %s CLASS(time)
  should: %s == Time.parse("2012-06-14 21:27:09 UTC")

#expires_on
  should: %s CLASS(time)
  should: %s == Time.parse("2017-01-26 00:00:00 UTC")


#registrar
  should: %s CLASS(registrar)
  should: %s.id           == "R2338-PRO"
  should: %s.name         == "Corporation Service Company dba CSC Corporate Domains, Inc"
  should: %s.organization == "Corporation Service Company dba CSC Corporate Domains, Inc"

#registrant_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type          == Whois::Record::Contact::TYPE_REGISTRANT
  should: %s[0].id            == "RSC-1"
  should: %s[0].name          == "Afilias Limited"
  should: %s[0].organization  == "Afilias Limited"
  should: %s[0].address       == "2 La Touche House\nIFSC"
  should: %s[0].city          == "Dublin"
  should: %s[0].zip           == "1"
  should: %s[0].state         == "IL"
  should: %s[0].country_code  == "IE"
  should: %s[0].phone         == "+353.14310511"
  should: %s[0].fax           == "+353.14310557"
  should: %s[0].email         == "domainadmin@afilias.info"

#admin_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type          == Whois::Record::Contact::TYPE_ADMIN
  should: %s[0].id            == "RSC-1"
  should: %s[0].name          == "Afilias Limited"
  should: %s[0].organization  == "Afilias Limited"
  should: %s[0].address       == "2 La Touche House\nIFSC"
  should: %s[0].city          == "Dublin"
  should: %s[0].zip           == "1"
  should: %s[0].state         == "IL"
  should: %s[0].country_code  == "IE"
  should: %s[0].phone         == "+353.14310511"
  should: %s[0].fax           == "+353.14310557"
  should: %s[0].email         == "domainadmin@afilias.info"

#technical_contacts
  should: %s CLASS(array)
  should: %s SIZE(1)
  should: %s[0] CLASS(contact)
  should: %s[0].type          == Whois::Record::Contact::TYPE_TECHNICAL
  should: %s[0].id            == "RSC-1"
  should: %s[0].name          == "Afilias Limited"
  should: %s[0].organization  == "Afilias Limited"
  should: %s[0].address       == "2 La Touche House\nIFSC"
  should: %s[0].city          == "Dublin"
  should: %s[0].zip           == "1"
  should: %s[0].state         == "IL"
  should: %s[0].country_code  == "IE"
  should: %s[0].phone         == "+353.14310511"
  should: %s[0].fax           == "+353.14310557"
  should: %s[0].email         == "domainadmin@afilias.info"


#nameservers
  should: %s CLASS(array)
  should: %s SIZE(5)
  should: %s[0] CLASS(nameserver)
  should: %s[0].name == "ns1.ams1.afilias-nst.info"
  should: %s[1] CLASS(nameserver)
  should: %s[1].name == "ns1.mia1.afilias-nst.info"
  should: %s[2] CLASS(nameserver)
  should: %s[2].name == "ns1.sea1.afilias-nst.info"
  should: %s[3] CLASS(nameserver)
  should: %s[3].name == "ns1.yyz1.afilias-nst.info"
  should: %s[4] CLASS(nameserver)
  should: %s[4].name == "ns1.hkg1.afilias-nst.info"
